resources/assets/less/jquery-ui/slider.less

```diff
@@ -0,0 +1,48 @@
+/*!
+ * jQuery UI Slider 1.8.23
+ *
+ * Copyright 2012, AUTHORS.txt (http://jqueryui.com/about)
+ * Dual licensed under the MIT or GPL Version 2 licenses.
+ * http://jquery.org/license
+ *
+ * http://docs.jquery.com/UI/Slider#theming
+ */
+
+.ui-corner-all {
```

Comment:
looks like it's part of base, not slider. Also there's `@border-radius-large` and stuff.